policy/approval/approve.go

```diff
@@ -103,9 +103,17 @@ func (r *Rule) Evaluate(ctx context.Context, prctx pull.Context) (res common.Res
 	res.Name = r.Name
 	res.Description = r.Description
 	res.Status = common.StatusSkipped
+	res.Requires = common.Actors{
+		Organizations: r.Requires.Organizations,
+		Teams:         r.Requires.Teams,
+		Users:         r.Requires.Users,
+	}
```

Comment:
```suggestion
	res.Requires = r.Requires.Actors
```

Embedded fields in Go are implicitly named after their type, so can use this to copy the whole field in one operation.